Week1/homework/app.js

```diff
@@ -2,10 +2,128 @@
 
 {
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'So_Long_A_Letter',
+    'Macbeth',
+    'Julius_Caesar',
+    'Silas_Marner',
+    'The_Gods_Are_Not_To_Blame',
+    'Things_Fall_Apart',
+    'Database_Processing',
+    'JavaScript_For_Web_Developers',
+    'Installing_And_Configuring_Windows_10',
+    'Installing_And_Configuring_Windows_Server_2012_R2'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  function createList() {
+    const ul = document.createElement('ul');
+    for (let i = 0; i < bookTitles.length; i++) {
+      const li = document.createElement('li');
+      ul.appendChild(li);
+      li.setAttribute('id', bookTitles[i]);
+    }
+  }
+  window.onload = createList;
```

Comment:
You can assign only a single value to `window.onload`. The last value you assign overrides earlier values. Therefore the only function that gets executed is the `displayBooks` function, because the last assignment to `window.onload` is in line 128.